src/modules/m_nick.c

```diff
@@ -298,13 +298,13 @@ void nick_identify (Nick *nptr, User *uptr __unused, char *all)
     if (!user) {
         NoticeToUser(nptr,"You are not registered");
         return;
-    }   
-    
-    if (nptr->svid != NULL) {
+    }
+    if (nptr->svid[0] != '\0')
+    {
```

Comment:
Nit: the coding style does not do a newline before the opening bracket in branches (as opposed to functions).